NEXT_CHANGELOG.md

```diff
@@ -87,6 +87,24 @@ telemetry:
 
 By [@o0Ignition0o](https://github.com/o0Ignition0o) in https://github.com/apollographql/router/pull/1718
 
+### Dedicated health check endpoint removed with new recommendation to use `/query={__typename}` query ([Issue #1765](https://github.com/apollographql/router/issues/1765))
+
+We have *removed* the dedicated health check endpoint and now recommend users to configure their health checks (in, e.g, Kubernetes, Docker) to use a simple GraphQL query instead.
+
+Use the following query with a `content-type: application/json` heaader as a health check instead of `/.well-known/apollo/server-health`:
```

Comment:
```suggestion
Use the following query with a `content-type: application/json` header as a health check instead of `/.well-known/apollo/server-health`:
```